test_botframework.py

```diff
@@ -0,0 +1,76 @@
+from errbot import Message
+
+from botframework import BotFramework, Identifier, Conversation
+from test_common import mock_config
+from unittest.mock import MagicMock, call, patch
+import pytest
+
+member_id = 1
+member_name = 'member name'
+member_email = 'member@email.com'
+conversation_id = 1
+default_reply = {
+    'type': 'message',
+    'text': 'My text message'
+}
```

Comment:
this is should be fixture

Reply:
Honestly we don't think fixtures are really needed here. As one downside, you'd end up with very long functions with a bunch of parameters. As in many other programing languages, we think that just a variable would be enough here.